software/bifroest.yml

```
@@ -0,0 +1,12 @@
+name: Engity's Bifröst
+website_url: https://bifroest.engity.org/
+description: Highly customizable SSH server with several ways to authorize a user and options where and how to execute a user's session.
+licenses:
+  - Apache-2.0
+platforms:
+  - Go
+  - Docker
+tags:
+  - Remote Access
+  - Proxy
```

```suggestion
tags:
  - Remote Access
```

Not relevant, see tag description

> A [proxy](https://en.wikipedia.org/wiki/Proxy_server) is a server application that acts as an intermediary between a client requesting a resource and the server providing that resource. This section about forward (i.e. outgoing) proxies.

---

> This section about forward (i.e. outgoing) proxies.


My comment still stands. OpenSSH also has forwarding/proxy-like capabilities but it still doesn't count as a proxy in the context of this list.

---

@nodiscc updated as requested.